OrbitClientGgp/main.cpp

```diff
@@ -14,6 +14,7 @@ ABSL_FLAG(int32_t, pid, 0, "pid to capture");
 ABSL_FLAG(uint32_t, capture_length, 10, "duration of capture in seconds");
 ABSL_FLAG(std::vector<std::string>, functions, {},
           "Comma-separated list of functions to hook to the capture");
+ABSL_FLAG(std::string, file_name, "", "Set file name for saving the capture.");
```

Comment:
The "set" in the comment could suggest that this is a sticky setting. I would simply say:

"File name used for saving the capture."

Reply:
That makes sense, I updated it. Thanks!